### DISCUSSION TITLE
SMA overload function

### DISCUSSION THREAD

**Discussion by bitwizardry** [DaveSkender/Stock.Indicators#505] [Ideas]
Hi Dave, 

What about a GetSMA's overload with a `Func<IQuote, Decimal>`  allowing us to get the sma from any part of the candle, not only from the Close price? I think it's a nice idea.

**Comment by DaveSkender** [comment]
Lemme think about it.  I’ve seen this done in some charting platforms, but never quite understood why anyone would want to do that.  For the sake of avoiding superfluous features, I never tried.  What’s the reason someone would want an indicator on Low price, for example?

**Reply by bitwizardry** [reply]
Yes, my trading platform has this feature.  One of the reasons is to combine 2 or more SMAs, for example, if we plot the 2 SMAs based on High and Low,, we get a small range of prices, which can be used as support/resistence, just like bollinger bands, without volatility.

**Reply by DaveSkender** [reply]
I can see that.  I can also potentially see an entirely new channel style indicator with HL2 as centerline.

**Reply by DaveSkender** [reply]
Tracking issue: #506
